python/paddle/vision/ops.py

```diff
@@ -2017,7 +2017,7 @@ def _nms(boxes, iou_threshold):
     if top_k is None:
         return keep_boxes_idxs[sorted_sub_indices]
 
-    if in_dygraph_mode():
+    if in_dynamic_or_pir_mode():
```

Comment:
此处动态图逻辑和静态图逻辑无法统一到一起，请复原
```suggestion
    if in_dynamic_mode():
```